manifest.json

```
@@ -3,10 +3,10 @@
   "id": "ttrss",
   "packaging_format": 1,
   "description": {
-    "en": "A PHP and Ajax feed reader",
-    "fr": "Un lecteur de flux en PHP et Ajax"
+    "en": "A RSS feed reader",
```

```suggestion
    "en": "A news feed (RSS/Atom) reader and aggregator",
```

from https://tt-rss.org/